bindings/swift/native/make_framework.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -e
+
+while [ $# -gt 0 ]; do
+	if [[ $1 == *"--"* ]]; then
+        v="${1/--/}"
+        declare $v="$2"
+	fi
+	shift
+done
+
+if [ -z "$target" ]; then 
+	echo "--target is missing"
+	exit 1
+fi
+
+FRAMEWORK_NAME=IOTAWalletInternal
+FRAMEWORK_VERSION=1.0.0
+
+makeFramework() {
+	cd $1
+	rm -rf $FRAMEWORK_NAME.framework
+	mkdir $FRAMEWORK_NAME.framework
+	mkdir $FRAMEWORK_NAME.framework/Headers
+	mkdir $FRAMEWORK_NAME.framework/Modules
+
+	touch $FRAMEWORK_NAME.framework/Modules/Modules.modulemap
+	tee -a $FRAMEWORK_NAME.framework/Modules/Modules.modulemap > /dev/null <<EOT
+framework module $FRAMEWORK_NAME {
+  umbrella header "$FRAMEWORK_NAME.h"
+
+  export *
+  module * { export * }
+}
+EOT
```

Comment:
```suggestion
	cat > $FRAMEWORK_NAME.framework/Modules/Modules.modulemap << EOF
framework module $FRAMEWORK_NAME {
  umbrella header "$FRAMEWORK_NAME.h"

  export *
  module * { export * }
}
EOF
```

Reply:
It's not the same behaviour unfortunately

Reply:
`cat >>` instead of `cat >` then.